sdk/tables/data-tables/src/utils/connectionString.ts

```diff
@@ -6,6 +6,9 @@ import { fromAccountConnectionString, getAccountConnectionString } from "./accou
 import { ClientParamsFromConnectionString, ConnectionString } from "./internalModels";
 import { URL } from "./url";
 
+const DevelopmentConnectionString =
+  "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1";
```

Comment:
This is a publicly available connection string used by Azurite and Azure Storage Emulator

References:
https://github.com/Azure/Azurite/tree/table#default-storage-account
https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator#connect-to-the-emulator-account-using-the-shortcut

Reply:
The emulator may not be hosted locally, in which case this will fail. https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite#command-line-options

Reply:
It may also have a differently configured account name and key https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite#authorization-for-tools-and-sdks

Reply:
Yeah, it looks like in those more advanced scenarios customers may want to use an actual connection string instead of the development shortcut.

We will need to document this in the samples to make sure there is no confusion.

Makes sense? Or do you have something different in mind?

Reply:
I'm testing this right now but I think the shortcut points to different targets based on environment configuration. For example, if `AZURITE_ACCOUNTS` exists, it may change the target of the shortcut (becoming unaligned with your hardcoded value) and produce a confusing credential mismatch error when the user hasn't specified any credentials.

Of course in both scenarios, documentation could mitigate the impact so perhaps this isn't a blocker.

Reply:
Agreed.